backend/uclapi/timetable/management/commands/update_gencache.py

```diff
@@ -22,6 +22,10 @@
     Room, RoomA, RoomB, \
     Booking, BookingA, BookingB
 from common.helpers import LOCAL_TIMEZONE
+
+from common.cachet import create_incident, delete_incident, CachetException, \
+    get_incident_name
```

Comment:
Ughhh, it's annoying we hit the 80 line mark here. Looks a bit awkward, guess good opportunity for [PEP8 compliance](https://stackoverflow.com/questions/29534027/why-import-statements-with-parentheses) (can Hound do it's job please???)

Reply:
I don't believe we should follow PEP8 here, since most tool-chains will complain, if you try do the brackets instead of the backslash in an IDEA IDE for example like pycharm you will get an error.

:shrug: ?